python/cuml/test/dask/conftest.py

```diff
@@ -1,9 +1,10 @@
 import pytest
+
 from dask_cuda import LocalCUDACluster
 
 
 @pytest.fixture(scope="module")
 def cluster():
-    cluster = LocalCUDACluster()
+    cluster = LocalCUDACluster(protocol="ucx", threads_per_worker=1)
```

Comment:
Interesting, is there a downside to always using UCX? Are there users who will need to be stuck on TCP? Seems like it would be nice to make this optional so we can add a few non-UCX tests to ensure we don't break that path.

Reply:
That's an interesting observation. I've been approaching this assuming the Dask project will be responsible for making sure its objects are properly serialized when being communicated across the workers. Are you saying that, given our history with supporting new Dask releases, you'd prefer for us to test to both just to be absolutely certain?

Reply:
Yes, I was thinking some testing with each would be helpful. I know some internal users are running with UCX disabled on purpose. Not sure we need to run all tests both ways if it's slow but at least some fallback tests to make sure we don't break tcp would be nice. I'm happy to wait for a future change though.

Reply:
One benefit to our design in cuML is that the Dask cluster doesn't require UCX to be enabled in order to run cuML algorithms that use UCX. 

What do you think about having a ucx-enabled Dask fixture and non-ucx-enabled Dask fixture and just running the ucx-enabled fixture only for the mnmg algorithms which require UCX for now? This doesn't test both paths in every test, but it does at least get some coverage with both options. We could enable all the tests to use both code paths, but that could result in twice the runtime. I'll think about it a little bit more.

Reply:
Yes, I think that's basically what I was hoping for... simple and just needs to fixtures.

Reply:
As per my recent comment in https://github.com/rapidsai/cuml/pull/1405#issuecomment-562302319, that would be the preferred way for using `LocalCUDACluster` now.